server/models/commentModel.js

```diff
@@ -0,0 +1,16 @@
+import mongoose from 'mongoose';
+import autoIncrementId from './autoIncrementModel';
+
+const CommentSchema = new mongoose.Schema({
+  description: { type: String, required: true },
+  createdBy: { type: String, required: true },
+
+}, { timestamps: { createdAt: 'createdAt' } });
+
+async function prepProcess(next) {
```

Comment:
Abstract this function, because it's being used in multiple models.

Reply:
I do not find any advantage of abstraction here as each model implements it in its own way. Let me know your thought process on it.